src/App.jsx

```diff
@@ -10,6 +10,10 @@ import theme from './theme';
 import Main from './Main';
 
 axios.interceptors.request.use(config => {
+  if (config.url.startsWith('http')) {
```

Comment:
I assume this is to avoid sending credentials to Google and/or Auth0. Would it be better to check for a specific domain(s) instead of a protocol? Looking for just `http` feels like it's bound to break in the future. Or perhaps this is an argument for using Axios instances instead of a global? The latter might be out of scope for this PR.

Reply:
I think we never want to send the access token nor set `result.baseURL = BASE_URL;` when we are not sending a request to `BASE_URL`. I added a patch to add all of this logic inside a conditional:

```jsx
axios.interceptors.request.use(config => {
  const result = config;

  if (!config.url.startsWith('http')) {
    // set Authorization header
    // set baseURL
  }

  return result;
});
```

I think this will make it resilient to any future additions to that function. Let me know if you disagree. Happy to discuss.

Reply:
Ah, I think I get it now. Any Balrog backend URLs will only have a path component in them, whereas others will be a full URL.

Reply:
Yep.